examples/training/classification/densenet/imagenet/basic_training.py

```diff
@@ -11,26 +11,27 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
you can also remove the copyright header now, I disabled the lint check for examples.  Tutobooks doesn't work with this :(